examples/https_client/https_client.cc

```diff
@@ -0,0 +1,184 @@
+// Create an https client test program
+// Note this that example show how to run HTTPS client
+// which connect to the remote server, read a resource and print thi
+// the response to the LOG.
+// To see command line args run with --helpful command line flag
+// Since there is no reasonable default host name you must at minimum run
+// this with --host <domain name: https://en.wikipedia.org/wiki/Domain_name> such as www.google.com
+// (note that you should not add the service name at the start such as https).
+// To demo that this is not a blocking HTTPS client this would run 2 fibers for connection
+// and another task that basically does nothing.
+// output from this is either an error (in case we failed - try given illegal host or port) or a
+// the headers and the body from the host and resource you gave.
+// for example:
+// ./https_client --host redis.io --resource /commands/asking/ --vmodule=https_client=2
+// --logbuflevel=-1 --alsologtostderr
+// will print the content from https://redis.io/commands/asking command to the log as well as to the
+// terminal
+#include <openssl/err.h>
+
+#include <boost/beast/http.hpp>
+#include <chrono>
+#include <iostream>
+#include <memory>
+
+#include "base/flags.h"
+#include "base/init.h"
+#include "util/fibers/event_count.h"
+#include "util/http/http_client.h"
+#include "util/proactor_base.h"
+#include "util/proactor_pool.h"
+#include "util/tls/tls_engine.h"
+#include "util/tls/tls_socket.h"
+#include "util/uring/uring_pool.h"
+
+using util::ProactorBase;
+using util::ProactorPool;
+using util::http::Client;
+using util::http::SecureClient;
+namespace h2 = boost::beast::http;
+using ResponseType = h2::response<h2::string_body>;
+using absl::GetFlag;
+
+ABSL_FLAG(std::string, http_port, "443", "HTTPS remote server port.");
+ABSL_FLAG(std::string, host, "", "Remote host domain name;
```

Comment:
did you try to compile it?

Reply:
sure - I was using this all the time to verify that its working

Reply:
I see what the problem, I did some cleanup before the push and remove the end of the flag, I'll fix it

Reply:
it was a retorical question :)
This version doesn't  compile. You can see it on the screen, it lacks parenthesis.

Reply:
and you can see the color of the text below. it should raise red flags.